devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/qute-example/java/src/main/java/org/acme/qute/Box.java

```diff
@@ -0,0 +1,20 @@
+package org.acme.qute;
+
+public class Box {
+
+    public enum Shape { SQUARE, CIRCLE }
```

Comment:
this reads weird - we have a enum named shape inside a class called Box which always is referred to as a either square or circle....

Maybe call Box a figure which has specific shape and size.